packages/dashboards/src/lib/configurator/components/formatters/link-formatter/link-formatter.component.ts

```diff
@@ -15,13 +15,16 @@ export class LinkFormatterComponent implements OnChanges, IHasChangeDetector {
     static lateLoadKey = "LinkFormatterComponent";
 
     @Input() data: ILinkFormatterData;
+    @Input() target?: boolean;
 
+    public targetSelf: string;
     public isValid = false;
 
     constructor(public changeDetector: ChangeDetectorRef) {
     }
 
-    ngOnChanges(changes: SimpleChanges) {
+    public ngOnChanges(changes: SimpleChanges) {
         this.isValid = Boolean(this.data && this.data.value && this.data.link);
+        this.targetSelf = this.target ? "_self" : "_blank";
```

Comment:
do we need to keep it in targetSelf?

we could do:
````
@Input() target = false;
and in template
[target]="target ? '_self' ? '_blank'"
````

Reply:
Both have the same behavior, i decided to do it there.